catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -265,8 +274,12 @@ public void addFields(Metacard metacard, SolrInputDocument solrInputDocument)
                     // Prevent adding a field already on document
                     if (solrInputDocument.getFieldValue(formatIndexName) == null) {
                         solrInputDocument.addField(formatIndexName, attributeValues);
-                        if (!(formatIndexName.endsWith(SchemaFields.BINARY_SUFFIX)
-                                || formatIndexName.endsWith(SchemaFields.OBJECT_SUFFIX))) {
+                        if (AttributeFormat.GEOMETRY.equals(format)) {
+                            solrInputDocument.addField(
+                                    formatIndexName + SchemaFields.SORT_KEY_SUFFIX,
+                                    createCenterPoint(attributeValues));
```

Comment:
There is a trade off between the performance increase and the decrease in accuracy with larger geometries which might be sorted further away with this change.  

>RPT offers more features than LatLonType and fast filter performance, although LatLonType is more appropriate when efficient distance sorting/boosting is desired. They can both be used simultaneously for what each does best – LatLonType for sorting/boosting, RPT for filtering. 
https://cwiki.apache.org/confluence/display/solr/Spatial+Search

Hopefully we will be able to go back to full geometry sorting after multi-value distance sort for RPT is optimized.

I do not have numbers yet for ingest performance but I expect it to be as fast or faster since the LatLon index is smaller than the previous geo sort key.